extensions/resource-deployment/src/ui/deployClusterWizard/deployClusterWizard.ts

```diff
@@ -82,8 +91,9 @@ export class DeployClusterWizard extends WizardBase<DeployClusterWizard, WizardP
 		await this.scriptToNotebook();
 	}
 
-	private getPages(): WizardPageBase<DeployClusterWizard>[] {
-		const pages: WizardPageBase<DeployClusterWizard>[] = [];
+	private getPages(): WizardPageBase<DeployClusterWizard, DeployClusterWizardModel>[] {
+		const pages: WizardPageBase<DeployClusterWizard, DeployClusterWizardModel>[] = [];
```

Comment:
you can combine this and the next line thusly. In general, we should strive to initialize variable to valid values and avoid, undefined, null, empty string, and empty array where possible.

const pages: WizardPageBase<DeployClusterWizard, DeployClusterWizardModel>[] = [];